crates/nu-command/src/system/run_external.rs

```diff
@@ -85,9 +85,34 @@ impl Command for External {
             return Ok(PipelineData::new(call.head));
         }
 
+        let mut args_strs = vec![];
```

Comment:
I would suggest to use Vec::with_capacity

Reply:
It might be hard to figure out the correct capacity beforehand. You'd need to first unwrap all the Values and at that point you can just collect them already.